docs/topics/install/upgrade-to-edge-stack.md

```diff
@@ -1,26 +1,21 @@
 # Upgrade to the Ambassador Edge Stack
 
-If you currently have the open source API Gateway version of Ambassador, you can upgrade to the Ambassador Edge Stack with a few simple commands. When you upgrade to the Ambassador Edge Stack, you'll be able to access several new features, such as **automatic HTTPS/TLS termination, Swagger/OpenAPI support, API catalog, IdP support, the Edge Policy Console (a declarative interface), and many more.**
+If you currently have the open source API Gateway version of Ambassador, you can upgrade to the Ambassador Edge Stack with a few simple commands. When you upgrade to the Ambassador Edge Stack, you'll be able to access additional capabilities such as **automatic HTTPS/TLS termination, Swagger/OpenAPI support, API catalog, Single Sign-On, the Edge Policy Console UI, and more.** For more about the differences between Edge Stack and the API Gateway, see https://www.getambassador.io/editions.
```

Comment:
The link to the editions page should be relative. In other places we've hyperlinked [editions page] to (/editions).